modules/affile/affile-dest.c

```diff
@@ -97,7 +97,7 @@ struct _AFFileDestWriter
   time_t last_msg_stamp;
   time_t last_open_stamp;
   time_t time_reopen;
-  struct iv_timer reap_timer;
+  struct iv_timer retry_timer;
```

Comment:
We should keep the reap_timer and should add a new reopen_timer, using one timer with 2 responsibility is very confusing, I think.
Theoretically these two timers (reap_timer and reopen_timer) will never registered at same time.
So I think, using one timer for reap and another timer for reopen, would be a better solution.